.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Enhancement
+about: For feature requests and discussion of ideas
+title: Enhancement
+labels: 'enhancement'
+assignees: ''
+
+---
+
+Page: <!--- If applicable, for instance "http://localhost:9090/system" -->
```

Comment:
Sorry to comment on this one again, but I prefer if people just mentioned the name of the page ("Network", "Machines").

Reply:
https://github.com/cockpit-project/cockpit/pull/11042#discussion_r250276080 

I'm fine with either :smiley_cat:

Reply:
Page name also WFM. It's usually clear from the description, so I almost feel it's redundant. Specific URL would be more helpful if you are on a particular subpage. But let's not block the PR on this, we can always change it later. So "Page:" it is.